sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2736,20 +2736,6 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
-  val LEGACY_PARQUET_REBASE_MODE_IN_WRITE =
```

Comment:
I moved the configs to avoid the errors:
```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala:2749:13: Reference to uninitialized value LEGACY_PARQUET_INT96_REBASE_MODE_IN_WRITE
[error]         s"${LEGACY_PARQUET_INT96_REBASE_MODE_IN_WRITE.key}.")
[error]             ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala:2781:13: Reference to uninitialized value LEGACY_PARQUET_INT96_REBASE_MODE_IN_READ
[error]         s"${LEGACY_PARQUET_INT96_REBASE_MODE_IN_READ.key}.")
[error]             ^
[error] two errors found
```